karrot/activities/api.py

```diff
@@ -281,11 +294,29 @@ def ics_detail(self, request, pk=None):
         renderer_classes=(ICSCalendarRenderer, ),
         serializer_class=ActivityICSSerializer,
         url_path='ics',
-        authentication_classes=[BasicAuthentication, SessionAuthentication],
+        authentication_classes=[BasicAuthentication, SessionAuthentication, ICSQueryTokenAuthentication],
```

Comment:
I think this means it tries the other auth mechanisms before reading the token.

Reply:
It does. I just tried it, sending basic auth, and it uses that first, then the token.

Great it still works with basic auth :)